automation-tests/test/environmentSettings.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export class EnvironmentSettings {
+  private static _instance: EnvironmentSettings;
+
+  private _devHubAliasName: string; // vscodeOrg
+  private _devHubUserName: string; // svc_idee_bot@salesforce.com
+  private _throttleFactor: number; // 1
+
+  private constructor() {
+  }
+
+  public static getInstance(): EnvironmentSettings {
+    if (!EnvironmentSettings._instance) {
+      EnvironmentSettings._instance = new EnvironmentSettings();
+
+      // Each setting is required, so preload them and assert if one is not found.
+      EnvironmentSettings._instance._devHubAliasName = process.env.DEV_HUB_ALIAS_NAME || 'vscodeOrg';
```

Comment:
user ?? for these assignments instead of || and you avoid the can't set zero or '' problem.  You can also set the defaults on the var declarations if you want.

Reply:
`private _devHubAliasName: string = 'vscodeOrg';` instead of 
`private _devHubAliasName: string; // vscodeOrg`

Reply:
I turned on strict mode in tsconfig and updated this and the other places strict mode yielded errors.